docs/pages/connect-your-client/putty.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Using PuTTY to connect to SSH nodes via Teleport
+description: This reference shows you how to use PuTTY to connect to SSH nodes via Teleport
+---
+
+This guide will show you how to use the Teleport client tool `tsh` to add saved sessions for use
+with [PuTTY](https://www.putty.org/), then use PuTTY as a client to connect to SSH nodes.
+
+You will learn how to:
+
+- Generate saved PuTTY sessions for SSH nodes attached to a Teleport cluster.
+- Log in to an interactive shell using these saved PuTTY sessions.
+
+
+## Prerequisites
+
+- A client machine running Windows 10 or higher (`tsh` support for adding saved sessions to PuTTY is only implemented for Windows)
+
+- The Teleport `tsh.exe` client version 14.0.0 or higher installed
+
+  ```code
+  $ curl.exe -O https://cdn.teleport.dev/teleport-v(=teleport.version=)-windows-amd64-bin.zip
+  # Unzip the archive and move `tsh.exe` to your %PATH%
+  ```
+- PuTTY for Windows version 0.78 or higher installed (you can [download PuTTY here](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html))
+
+
+## Summary
+
+To be able to add saved sessions to PuTTY, you will need an active `tsh` login.
+
+<ScopedBlock scope={["oss","enterprise"]}>
+
+```bash
+# Log into a Teleport cluster. This command retrieves the user's certificates
+# and saves them into %USERPROFILE%/.tsh/teleport.example.com
+C:\Users\gus>tsh login --proxy=teleport.example.com
+...
+
+# List SSH nodes that the user can connect to inside the cluster
+C:\Users\gus>tsh ls
+Node Name                           Address        Labels
+----------------------------------- -------------- ----------------------------
+ip-172-31-30-140                    127.0.0.1:3022 company=acmecorp,env=aws,...
+ip-172-31-34-128.us-east-2.compu... ⟵ Tunnel      access=open,enhanced_reco...
+
+
+# Add a saved session for the login 'ubuntu' on the node 'ip-172-31-30-140' to the Windows registry
+# If you do not provide a login to this command, your local Windows username will be used
+# instead ('gus' in this example)
+C:\Users\gus>tsh puttyconfig ubuntu@ip-172-31-30-140
+Added PuTTY session for ubuntu@ip-172-31-30-140 [proxy:teleport.example.com]
+```
```

Comment:
1. Add a saved session for a specific login on a specific node to the Windows registry. 

   For example, you can add a saved session for the login 'ubuntu' on the node 'ip-172-31-30-140' to the Windows registry by running the following command:
   
   ```code
   C:\Users\gus>tsh puttyconfig ubuntu@ip-172-31-30-140
   Added PuTTY session for ubuntu@ip-172-31-30-140 [proxy:teleport.example.com]
   ```

   If you don't provide a login to this command, your local Windows username is used instead.